_compdemos/cloud-faq.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
I think we should not put this FAQ up b/c as things evolve it will be hard to maintain.  I'm going to just take it out.  The main parts of this content that is still relevant are already discussed elsewhere.